tests/cases/fourslash/completionListInUnclosedFunction09.ts

```diff
@@ -5,5 +5,4 @@
 ////        var v = /*1*/
 ////}
 
-// Note: "v" questionable since we're in its initializer
-verify.completions({ marker: "1", includes: ["foo", "x", "y", "z", "bar", "a", "b", "c", "v"], isNewIdentifierLocation: true });
+verify.completions({ marker: "1", includes: ["foo", "x", "y", "z", "bar", "a", "b", "c"], isNewIdentifierLocation: true });

```

Comment:
Why didn't this work before? There was already code for it and you didn't change it very much.

Reply:
https://github.com/microsoft/TypeScript/pull/52690#issuecomment-1424655384 I guess thats because I'm now using `contextToken` instead of `location`, I didn't really dig into specific test cases, but isnt it an improvement? Also see https://github.com/microsoft/TypeScript/pull/52690#discussion_r1101825361